<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddVendorView</class>
 <widget class="QDialog" name="AddVendorView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Vendor Name</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="salesContactName">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Sales Contact Name</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="SalesContactPhone">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Sales Contact Phone</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="Address">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Company Address</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addVendor">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>200</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Vendor</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
